znap-cli/src/commands/new.rs

```diff
@@ -69,7 +69,7 @@ pub fn run(name: &str, dry_run: bool) {
             znap_toml_path.as_path(),
             &toml::to_string(&Config {
                 collections,
-                identity: "~/.config/solana/id.json".to_string(),
+                identity: None,
```

Comment:
This default value `"~/.config/solana/id.json"` is really nice because Solana CLI generates it in the same place. So all the tooling in Solana tends to rely on that. I think it's fair to say that the majority of users in local would prefer to have that as default.

Note: I'd still want the Env variable to have a higher precedence, so you can have it in the Toml but you can easily override it via an Env Variable.